src/style.css

```
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.woff2") format("woff2"),
+    url("./fonts/Avenir-Book.woff") format("woff"),
+    url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.woff2") format("woff2"),
+    url("./fonts/Avenir-Heavy.woff") format("woff"),
+    url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+.searchbar {
+  font-family: "Avenir-Heavy";
```

всегда добавляй дополнительные шрифты, если основной не загрузится
```suggestion
  font-family: "Avenir-Heavy", sans-serif;
```

---

не исправлено же

---

my bad, fixed